jekyll/_cci2_ja/server/overview/release-notes.adoc

```diff
@@ -1,19 +1,20 @@
 ---
 
-description: CircleCI Server 4.x の各リリースに含まれる新機能の詳細
-version:
-- Server v4.x
-- サーバー管理者
+contentTags:
+  platform:
+    - Server v4.x
+    - Server Admin
 ---
-= CircleCI Server v4.x のリリースノート
+= CircleCI server v4.x release notes
 :page-layout: classic-docs
 :page-liquid:
+:page-description: Details of the new features included in each CircleCI server 4.x release.
 :icons: font
 :toc: macro
 
 :toc-title:
 
-Server v4.x では:
```

Comment:
@yuki-sakamoto-san would it work to use the original here but change 4.x to 4.0?